grant-project/grant-monorepo
# Conditionally show "See more" button

### What This Does

Conditionally shows the "See more" button on proposal view, only if content is overflowing.

### Steps to Test

* Make a proposal with limited content, confirm it doesn't have the see more button
* Make a proposal with medium content, confirm it doesn't have the see more button unless you're at mobile size
* Make a proposal with a lot of content, confirm it has the see more button all the time
